seaice/pipeline/many_rasters_both_regions.py

```diff
@@ -0,0 +1,123 @@
+"""The script creates a unified Chukchi + Beaufort dataset for cases where both regions have data from one or more rasters. The output is a merged raster that uses the time series maximum of the data from each region. This is considered Case D: many rasters, both regions."""
+
+import argparse
+import pickle
+import rasterio as rio
+import pandas as pd
+import numpy as np
+from pathlib import Path
+from rasterio.merge import merge
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Execute raster unification Case D.")
```

Comment:
This miiiiight be able to be refactored out from the 3 scripts that use it, but it could also be messy -- skip for now, but in the future, smell for duplication similar to this and see if it can be simplified easily.  Harmless here tho.

Reply:
There are some different arg flavors coming to each function - so not quite enough redundancy to refactor, but DRY is noted!